subcommands/targets/deltas.go

```diff
@@ -0,0 +1,109 @@
+package targets
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	byTag  string
+	dryRun bool
+)
+
+func init() {
+	var deltas = &cobra.Command{
+		Use:   "static-deltas <target-version> [<from-version>...]",
+		Short: "Generate static deltas to the given target version to make OTAs faster",
+		Run:   doDeltas,
+		Args:  cobra.MinimumNArgs(1),
+		Long: `In many cases OTAs will have many ostree changes. These OTAs can be 
+sped up significantly by generating OSTree static deltas. Static deltas
```

Comment:
`sped` --> `speed`

Reply:
no this is "sped up" as in made faster. maybe i should say "These updates can be downloaded faster by generating .."

Reply:
redid the wording - https://github.com/foundriesio/fioctl/pull/99/commits/abb98ca62cfc1ce83520149e69977c0ff7656ee8